exercises/reforestation/canonical-data.json

```
@@ -0,0 +1,93 @@
+{
+    "exercise": "reforestation",
+    "version": "1.0.0",
+    "cases": [
+      {
+        "description": "Base case: empty tree",
+        "property": "treeTraversals",
+        "input": {
+          "preorder": [],
+          "inorder": []
+        },
+        "expected": {}
+      },
+      {
+        "description": "1 more step than base case",
+        "property": "treeTraversals",
+        "input": {
+            "preorder": ["a"],
+            "inorder": ["a"]
+        },
+        "expected": {"v":"a", "l": {}, "r": {}}
+      },
+      {
+        "description": "Core functionality w/ realistic size traversal",
```

```suggestion
        "description": "Core functionality with realistic size traversal",
```
This may make things a little easier for some tracks using test generators to not have to cope with abbreviations.